netci.groovy

```diff
@@ -14,17 +14,10 @@ def windowsUnitTestMachine = 'win2016-base'
 
 static void addRoslynJob(def myJob, String jobName, String branchName, Boolean isPr, String triggerPhraseExtra, Boolean triggerPhraseOnly = false) {
   def archiveSettings = new ArchivalSettings()
-  archiveSettings.addFiles('Binaries/**/*.pdb')
-  archiveSettings.addFiles('Binaries/**/*.xml')
-  archiveSettings.addFiles('Binaries/**/*.log')
-  archiveSettings.addFiles('Binaries/**/*.dmp')
-  archiveSettings.addFiles('Binaries/**/*.zip')
+  archiveSettings.addFiles('Binaries/Debug/Logs/**/*)
+  archiveSettings.addFiles('Binaries/Release/Logs/**/*)
```

Comment:
Not 100% sure this is a valid glob in groovy but fairly certain.

Reply:
I believe you actually just want `Binaries/Release/Logs/**`, looking at the examples here: https://github.com/dotnet/dotnet-ci/blob/ec853cba3865791cc59a84f386bdfd9091331ef1/docs/CI-CLASSIC.md#buildjob-construction-functionality

Reply:
@jaredpar, why not `Binaries/*/Logs/**/*`?